internal/core/drand_test.go

```diff
@@ -309,7 +309,8 @@ func TestRunDKGReshareAbsentNodeDuringExecution(t *testing.T) {
 			t.Logf("Node %d stopped \n", nodeIndexToStop)
 		},
 	}
-	newGroup, err := dt.RunReshareWithHooks(t, dt.clock.Now().Add(3*beaconPeriod), currentNodes, newNodes, hooks)
+	dt.thr = 3
```

Comment:
Why this change? Isn't that set in `NewDrandTestScenario`? 
Also, why not use one of the threshold variable declared above?

Reply:
you pass it into the `NewDrandTestScenario`, but it doesn't get updated for reshares

Reply:
Would the proper fix here be to adapt the threshold in the `RunReshareWithHooks` function?

Reply:
done